SCClassLibrary/Common/GUI/Base/QWebView.sc

```diff
@@ -253,6 +253,27 @@ WebView : View {
 	contentsSize { ^this.getProperty('contentsSize') }
 
 	scrollPosition { ^this.getProperty('scrollPosition') }
+    scrollPosition_ { |point| ^this.setProperty('scrollPosition', (point / this.zoom)) }
```

Comment:
Please use tabs instead of spaces when indenting .sc files. Note: I always mess this up myself :)

Reply:
done.